src/main/java/org/osiam/client/AuthService.java

```diff
@@ -50,13 +62,18 @@
  */
 class AuthService {
 
+    private static final String AUTHORIZATION_HEADER = "Authorization";
     private static final String BEARER = "Bearer ";
     private static final String TOKEN_ENDPOINT = "/oauth/token";
+    private static final String TOKEN_REVOCATION_ENDPOINT = "/token/revocation";
+    private static final String TOKEN_VALIDATION_ENDPOINT = "/token/validation";
 
     private final String endpoint;
     private final String clientId;
     private final String clientSecret;
     private final String clientRedirectUri;
+    private final int connectionTimeout;
+    private final int readTimeout;
```

Comment:
If we leave the initialization of them like that, they default to null (0) As is the default of jersey. Is this intentional? If so it should be documented in the Javadoc.